modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_requests_spec.rb

```diff
@@ -101,7 +104,189 @@
               ].join(' ')
             },
             'resolution' => nil
+          }
+        ]
+      )
+    end
+
+    it 'returns the list of accepted power of attorney requests and orders them correctly by poa_request.created_at' do
+      poa_requests
+
+      poa_requests[1].created_at = time_plus_one_day
+      poa_requests[1].save
+
+      get '/accredited_representative_portal/v0/power_of_attorney_requests', params: { status: 'Accepted' }
+
+      parsed_response = JSON.parse(response.body)
+      expect(response).to have_http_status(:ok)
+
+      expect(parsed_response).to eq(
```

Comment:
Would it be cleaner to place this large response in a test helper or factory even?